config/default/core.entity_view_display.course_collections.instructor.default.yml

```diff
@@ -0,0 +1,123 @@
+uuid: 65e44979-6f86-4e60-af11-88c7f03aabb4
```

Comment:
This config acts as the 'sensible default' and is allowed to be changed in the UI?

Reply:
not on ECK entities.

Reply:
I recently locked down form and video display configs for everything _except_ nodes. paragraphs and ECK will be locked

Reply:
Ah cool. Good to know.